apps/workspaces/actions.py

```
@@ -10,7 +12,7 @@
 logger.level = logging.INFO
 
 
-def export_to_netsuite(workspace_id, export_mode=None, expense_group_ids=[]):
+def export_to_netsuite(workspace_id, export_mode=None, expense_group_ids=[], triggered_by: ExpenseImportSourceEnum = None):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)